test/data/utilities.test.ts

```diff
@@ -15,13 +15,17 @@ test('all authorities.json utilities are in DB', async t => {
     .map(Object.keys)
     .flat();
 
+  const query = await database.all<{ utility_id: string; ct: number }[]>(
+    'SELECT utility_id, COUNT(1) AS ct FROM zip_to_utility GROUP BY utility_id',
+  );
+  const utilityCounts = Object.fromEntries(
+    query!.map(({ utility_id, ct }) => [utility_id, ct]),
+  );
+
```

Comment:
This change isn't strictly necessary but this test had gotten super slow with all the states' utilities added. This speeds it up by doing one DB query instead of thousands